Unit.h

```diff
@@ -0,0 +1,24 @@
+#ifndef HEADERFILE_H
+#define HEADERFILE_H
+
+
+#include <string>
+#include <sstream>
+
+class Unit
+{
+private:
+    std::string Name;
+    int Health;
+    int Damage;
+public:
+    Unit(std::string pName,std::string pHealth, std::string pDamage);
+    ~Unit();
+    std::string getName();
+    int getHealth();
+    int getDamage();
+    void getAttacked(int amount);
+    void setHealth(int newHealth);
+};
```

Comment:
I find the lack of consts very disturbing.

Reply:
Illetve a setHealth tul altalanos, nem akarjuk, hogy akarki akarmikor akarmire beallithassa. Csak ha egy masik unit meguti, akkor annak a damage-evel csokkenjen.